update records query (#256)
decentralized-identity/decentralized-web-node: spec/spec.md

  - The object ****MUST**** contain a `messageTimestamp` property, and its value
    ****MUST**** be of type string property, and its value ****MUST**** be an
    [[spec:rfc3339]] ISO 8601 timestamp that ****MUST**** be set and interpreted

>     as the time the `RecordsWrite` was created by the DID owner or another permitted party.

Author: ```suggestion
    as the time the `RecordsRead` record itself was created by the requester.
```

- The message object ****MUST**** contain a `descriptor` property, and its value ****MUST**** be a JSON object composed as follows:
  - The object ****MUST**** contain an `interface` property, and its value ****MUST**** be the string `Records`.
  - The object ****MUST**** contain a `method` property, and its value ****MUST**** be the string `Query`.
  - The object ****MUST**** contain a `messageTimestamp` property, and its value
    ****MUST**** be of type string property, and its value ****MUST**** be an
    [[spec:rfc3339]] ISO 8601 timestamp that ****MUST**** be set and interpreted

>     as the time the `RecordsWrite` was created by the DID owner or another permitted party.

Author: ```suggestion
    as the time the `RecordsQuery` message was created by the requester.
```

  - The object ****MAY**** contain a `filter` property, and if present its value ****MUST**** be an object that ****MAY**** contain the following properties:
    - The object ****MAY**** contain a `attester` property representing the

>       creator of the record's did. If present its value ****Must**** be a string in the form of a did.

Author: ```suggestion
      creator of the `Record(s)` did. If present its value ****MUST**** be a string in the form of a DID.
```

    - The object ****MAY**** contain a `receipient` property representing the

>       recipient of the record's did If present its value ****Must**** be a string in the form of a did.

Author: ```suggestion
      recipient of the record's DID If present its value ****MUST**** be a string in the form of a DID.
```

    - The object ****MAY**** contain a `schema` property, and if present its value ****Must**** be a URI string that indicates the schema of the associated data.
    - The object ****MAY**** contain a `recordId` property, and its value ****MUST**** be a [_Computed Record ID_](#computed-record-ids).
    - The object ****MAY**** contain a `parentId` property, and if present its

>       value ****Must**** be a string that represents the computed record if of

Author: ```suggestion
      value ****MUST**** be a string that represents the computed record of
```

      the parent object.
    - The object ****MAY**** contain a `contextId` property, and its value ****MUST**** be the deterministic ID for a contextually linked set of objects.
    - The object ****MAY**** contain a `dateCreated` property. If present, it
      **MUST** include the `from` and `to` property described as a string range in the ISO 8601 format. 
      - The from properties value ****MUST**** be of type string property, and its value ****MUST**** be an [[spec:rfc3339]] ISO 8601 timestamp.
       - The to properties value ****MUST**** be of type string property, and its value ****MUST**** be an [[spec:rfc3339]] ISO 8601 timestamp.

>     - The object ****MAY**** contain a `protocol` property, and its value ****Must**** be a URI that denotes the Protocol an object is a part f.

Author: ```suggestion
    - The object ****MAY**** contain a `protocol` property, and its value ****MUST**** be a URI that denotes the Protocol an object is a part of.
```
